src/Apps/Order2/Routes/Checkout/Order2CheckoutRoute.tsx

```diff
@@ -37,9 +42,11 @@ export const Order2CheckoutRoute: React.FC<Order2CheckoutRouteProps> = ({
 
   return (
     <Analytics contextPageOwnerId={order.internalID}>
-      <Order2CheckoutContextProvider order={order}>
-        <Order2CheckoutApp order={order} />
-      </Order2CheckoutContextProvider>
+      <Elements stripe={stripePromise}>
+        <Order2CheckoutContextProvider order={order}>
+          <Order2CheckoutApp order={order} />
+        </Order2CheckoutContextProvider>
+      </Elements>
```

Comment:
I hoisted Stripe's Elements here to be able to use the `stripe` module anywhere in the checkout flow using  the `useStripe` hook